### PR TITLE
Implement gRPC streaming

### LINKED ISSUE #26: Implement streaming gRPC

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes #26

### CODE REVIEW

File: pkg/dispatcher/grpc/grpc.go [hunk 1]
```diff
@@ -19,37 +19,76 @@ package grpc
 import (
 	"google.golang.org/grpc"
 
-	function "github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/function"
-	fntypes "github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/fntypes"
+	"github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/function"
+	_ "github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/fntypes"
```
Comment (markfisher) [inline pkg/dispatcher/grpc/grpc.go]: why is this listed twice?

Comment (ericbottard) [reply pkg/dispatcher/grpc/grpc.go]: Not the same package.
I'm also not 100% sure it needs to be imported if not used (hence the `_`) but there is an `init` function in there that may be required

Comment (markfisher) [reply pkg/dispatcher/grpc/grpc.go]: I'm talking about `fntypes` being listed again 5 lines below

Comment (ericbottard) [reply pkg/dispatcher/grpc/grpc.go]: Ohhh... My bad

**Comment by markfisher** [comment]
I think we need to coordinate this PR merging with an update to the Python `uppercase` sample, which is currently the only one using gRPC: https://github.com/projectriff/riff/tree/master/samples/python/uppercase

cc @dturanski

**Comment by dturanski** [comment]
Yup. I'll take a look.

**Comment by dturanski** [comment]
Also, we `function-proto` is not updated. We should either change the sidecar to pull the schema from there or point function developers to the function-sidecar artifacts and remove `function-proto`.  This is a topic deserving of more discussion in light of BYO schema and/or RSocket support for sidecar, so I'm ok with removing `function-proto` for now.

**Comment by dturanski** [comment]
LGTM; merging

**Pull request closed without merging**